test/gui/lib/test_foldable.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+
+from randovania.gui.lib.foldable import Foldable
+
+
+def test_foldable_initial_state():
```

Comment:
Check the other tests that involve GUI. I think all you need is to add the `skip_qtbot` fixture, but might be a good idea to add your widget to the qtbot thingy so they're cleaned up.